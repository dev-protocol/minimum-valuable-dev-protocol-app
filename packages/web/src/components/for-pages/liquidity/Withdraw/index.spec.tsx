import React from 'react'
import { render } from '@testing-library/react'
import { Withdraw } from '.'
import 'src/__mocks__/window/matchMedia.mock'

jest.mock('src/pages/liquidity/fixtures/geyser/hooks.ts')
jest.mock('src/pages/liquidity/fixtures/uniswap-pool/hooks.ts')

describe(`${Withdraw.name}`, () => {
  test('Snapshot', () => {
    const component = render(<Withdraw />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
