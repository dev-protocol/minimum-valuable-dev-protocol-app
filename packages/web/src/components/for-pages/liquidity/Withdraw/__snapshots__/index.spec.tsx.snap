// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Withdraw Snapshot 1`] = `
<body>
  <div>
    <form
      class="ant-form ant-form-vertical"
    >
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-4swyw2-0 lpiJWU"
              >
                <label
                  for="amount"
                >
                  Your amount
                </label>
                <span
                  class="ant-input-affix-wrapper ant-input-affix-wrapper-lg sx3fo9-0 iSujTX"
                >
                  <input
                    class="ant-input ant-input-lg"
                    id="amount"
                    type="number"
                    value=""
                  />
                  <span
                    class="ant-input-suffix"
                  >
                    <span
                      class="sc-141ox4c-0 jHfpSs"
                    >
                      WETHDEV-V2
                    </span>
                    <button
                      class="ant-btn ant-btn-primary ant-btn-sm sc-1allgzb-0 jyemkU"
                      type="button"
                    >
                      <span>
                        Max
                      </span>
                    </button>
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <div
                class="sc-4swyw2-0 lpiJWU"
              >
                <label
                  for="estimated"
                >
                  Rewards claimed
                </label>
                <input
                  class="ant-input ant-input-lg ant-input-disabled sx3fo9-0 iSujTX"
                  disabled=""
                  id="estimated"
                  type="number"
                  value="0"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-row ant-form-item w1437o-0 fRMLzT"
      >
        <div
          class="ant-col ant-form-item-control"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <button
                class="ant-btn ant-btn-primary ant-btn-lg"
                type="button"
              >
                <span>
                  Withdraw
                </span>
              </button>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
